src/utils/hooks/useReferenceDocs.ts

```diff
@@ -38,7 +38,7 @@ export const useReferenceDocs = (currentUser: User | null) => {
 }
 
 const shouldSeeInternalDocs = (user: User): boolean => {
-  if (!user?.isAdmin) return true
+  if (!user?.isAdmin) return false
   if (!user?.organisation) return false
   return user.organisation.isSystemOrg
 }
```

Comment:
Needs the same fix on the "shouldSeeExternalDocs" below -- Admin should see all docs (That's how it was previously -- see the git diff I posted)